ocs_ci/ocs/resources/mcg.py

```diff
@@ -35,6 +37,10 @@ def __init__(self):
         """
         Constructor for the MCG class
         """
+
+        # To clean the logs from warnings related to the lack of SSL certs
```

Comment:
Can we address these warnings by generating and configuring correct certificate? I am not sure if dropping warnings about not using SSL is a customer scenario.
If we want to suppress these messages for now, I would suggest to create issue for this, todo comment into code and a warning log message that SSL warnings are suppressed.

Reply:
I talked with NooBaa, and it's possible to generate a certificate, and NB should even automatically pull and use it. 
The comment is inaccurate - it doesn't stem from a lack of a cert, but rather by an inability to verify it, because it's self-signed.
However, I honestly don't know how easy it is to perform this per-cluster, or whether it's even possible to automate this, or do it quickly enough (generating a cert that isn't self-signed).

I chose to drop the warnings in order to reduce the log size and make it cleaner. How may it affect our actual testing in regards to mimicking customer scenarios?

Reply:
Regarding CLI usage - we won't be using any local binary anymore. All commands will run on the binary inside the operator's pod.

Reply:
Ok, thank you.